changelog/_unreleased/2022-11-20-add-information-about-sales-channel-deletion.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 2022-11-20-Add-information-about-sales-channel-deletion
+issue: 
+author: Frederik Dengler
+author_email: frederikdengler@gmail.com
+author_github: Frederik Dengler
+___
+# Administration
+* Add more information if someone deletes a sales channel. Before it was confusing if an error appears, no message at all. And I was already redirected to the dashboard if the sales channel was not deleted yet
```

Comment:
```suggestion
* Added more information if a sales channel is deleted, and only redirect if the deletion was successfull.
```

Changelog entries have to begin with certain keywords, e.g. `Added`

And changelogs should only describe what you did, the why and how it was previously should be stated in the PR description or commit message